.github/workflows/fetch-licenses.yaml

```
@@ -0,0 +1,60 @@
+name: Fetch Licenses
+
+on:
+    workflow_dispatch:
+        inputs:
+            force_run:
+                description: 'Force run license extraction'
+                required: false
+                default: 'false'
+    schedule:
+        - cron: '0 0 * * *' # Runs at midnight ET
```

Thanks for the reminder.  I do want it to run around midnight ET to keep it out of main processing for a lot of timezones.  Unfortunately, there's no way to keep it out of business hours for all timezones.  H

```suggestion
        - cron: '5 4 * * *' # Runs at 0405 UTC (0605 CET, 0005 ET, 2105 PT)
```